lib/routes/sms.js

```diff
@@ -98,6 +101,53 @@ module.exports = (log, isA, error, config, customs, sms) => {
           return {}
         }
       }
+    },
+    {
+      method: 'GET',
+      path: '/sms/status',
+      config: {
+        auth: {
+          strategy: 'sessionToken'
+        }
+      },
+      handler (request, reply) {
+        log.begin('sms.status', request)
+
+        let isOk
+
+        return getStatus()
```

Comment:
The shortcut should be inverted actually, it's more likely that the `getGeoData` check will fail than the balance check.